include/tvm/relay/attrs/transform.h

```diff
@@ -146,11 +146,18 @@ struct GatherAttrs : public tvm::AttrsNode<GatherAttrs> {
 
 struct GatherNDAttrs : public tvm::AttrsNode<GatherNDAttrs> {
   Integer batch_dims;
+  Integer num_indices_per_tuple;
 
   TVM_DECLARE_ATTRS(GatherAttrs, "relay.attrs.GatherNDAttrs") {
     TVM_ATTR_FIELD(batch_dims).set_default(Integer(0)).describe("The number of batch dimensions.");
+    TVM_ATTR_FIELD(num_indices_per_tuple)
```

Comment:
I think this name is a little difficult to parse, would it make sense to instead call it something like `num_dims_per_index` or `index_rank`?

Reply:
Also I'd like to think that this attributes was added to workaround the current Relay limitation I described, ideally it should go away when we have a better representation of runtime shape in relay...

Reply:
yeah thats fair, I personally think `index_rank` makes a little more sense but am fine either way.

Reply:
Change to `index_rank`.